__tests__/app.test.js

```diff
@@ -48,6 +48,84 @@ describe("/api", () => {
     });
   });
   describe("/GET api/articles", () => {
+    test(" GET /api/articles/ returns an array", () => {
+      return request(app)
+        .get("/api/articles/")
+        .expect(200)
+        .then(({ body }) => {
+          const articles = body.articles;
+          expect(Array.isArray(articles)).toBe(true);
+        });
+    });
+    test(" GET /api/articles/ returns a status 200 and all articles information except for body", () => {
+      return request(app)
+        .get("/api/articles/")
+        .expect(200)
+        .then(({ body }) => {
+          const articles = body.articles;
+          expect(articles.length).toBe(13);
+          articles.forEach((article) => {
+            expect(article.hasOwnProperty("article_id")).toBe(true);
+            expect(article.hasOwnProperty("title")).toBe(true);
+            expect(article.hasOwnProperty("topic")).toBe(true);
+            expect(article.hasOwnProperty("author")).toBe(true);
+            expect(article.hasOwnProperty("created_at")).toBe(true);
+            expect(article.hasOwnProperty("votes")).toBe(true);
+            expect(article.hasOwnProperty("article_img_url")).toBe(true);
+            expect(article.hasOwnProperty("body")).toBe(false);
+          });
+        });
+    });
+    test(" GET /api/articles/ returns a status 200 and a comment_count property on all articles", () => {
+      return request(app)
+        .get("/api/articles/")
+        .expect(200)
+        .then(({ body }) => {
+          const articles = body.articles;
+          articles.forEach((article) => {
+            expect(article.hasOwnProperty("comment_count")).toBe(true);
+          });
+        });
+    });
+    test(" GET /api/articles/ returns correct comment count for each article", () => {
```

Comment:
I don't think we really need this test. As long as we're checking the length in the previous two tests 👍